crates/red_knot_test/src/parser.rs

```diff
@@ -349,26 +355,10 @@ impl<'s> Parser<'s> {
             return Err(anyhow::anyhow!("Unterminated code block at line {line}."));
         }
 
-        let mut config: FxHashMap<&'s str, &'s str> = FxHashMap::default();
-
-        if let Some(config_match) = captures.name("config") {
-            for item in config_match.as_str().split_whitespace() {
-                let mut parts = item.split('=');
-                let key = parts.next().unwrap();
-                let Some(val) = parts.next() else {
-                    return Err(anyhow::anyhow!("Invalid config item `{}`.", item));
-                };
-                if parts.next().is_some() {
-                    return Err(anyhow::anyhow!("Invalid config item `{}`.", item));
-                }
-                if config.insert(key, val).is_some() {
-                    return Err(anyhow::anyhow!("Duplicate config item `{}`.", item));
-                }
-            }
+        if captures.name("config").is_some() {
+            return Err(anyhow::anyhow!("Configs are no longer allowed."));
```

Comment:
I misunderstood this message completely at first. I thought `toml`-configs would not be allowed anymore for some reason. Maybe something like:

```suggestion
            return Err(anyhow::anyhow!("Trailing code-block metadata is not supported. Only the code block language can be specified."));
```